.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build:
+    working_directory: /temp
+    docker:  
+      - image: mono:latest
+    environment:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+      DOTNET_CLI_TELEMETRY_OUTPUT: 1
+    steps:
+      - checkout
+      - run: msbuild Analytics/Analytics.csproj /t:restore
+      - run: msbuild Analytics/Analytics.csproj /p:Configuration=Release
+      - run: msbuild Analytics.Net35/Analytics.Net35.csproj /t:restore
+      - run: msbuild Analytics.Net35/Analytics.Net35.csproj /p:Configuration=Release
+      - run: nuget pack Analytics.nuspec
+  test:
+    working_directory: /test
+    docker:  
+      - image: microsoft/dotnet:1.1.1-sdk
+    environment:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+      DOTNET_CLI_TELEMETRY_OUTPUT: 1
+    steps:
+      - checkout
+      - run: dotnet restore Test/Test.csproj
+      - run: dotnet test Test/Test.csproj
+  test_35:
+    working_directory: /test35
+    docker:  
+      - image: mono:latest
+    environment:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+      DOTNET_CLI_TELEMETRY_OUTPUT: 1
+    steps:
+      - checkout
+      - run: nuget restore
```

Comment:
Same as earlier comment.

Could we make all these run steps as Makefile targets (e.g. https://github.com/segmentio/analytics-ios/blob/dev/circle.yml#L20-L23).

This makes it easy to run the same commands as we do on CI locally as well.